I card/SubmittedAssignment/Kishor_Parde/preview.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./css/style.css">
+    <title>ID Card</title>
+</head>
+
+<body>
+    <!-- ID Card Wrapper -->
+    <div class="id-card-wrapper-front ">
+        <!-- Card Background Front Side -->
+        <div class="id-card-background">
+            <div class="id-dark-pink"></div>
+            <div class="id-pink"></div>
+            <div class="id-light-pink"></div>
+        </div>
+
+        <!-- Content of ID Card Front Side -->
+        <div class="id-card-content">
+            <!-- Logo -->
+            <div class="org-logo">
+                <img src="img/logo.png" alt="Logo">
+            </div>
+            
+            <!-- Photo -->
+            <div class="user-photo">
+                <img src="img/profile1.png" alt="Photo">
+            </div>
+
+            <!-- Name -->
+            <div class="user-name">Kishor <br>Parde</div>
+            
+            <!-- ID Number -->
+            <div class="id-number">ID: 123456</div>
+            
+            <!-- Job Title/Grade/Class -->
+            <div class="id-title">Jr.Software Engineer </div> <!-- or Grade: 10 -->
+
+            <!-- Organization/School/College Information -->
+            <!-- <div class="org-name">Apnasite It Services</div>
+            <div class="org-address">Kalewadi,Pune</div> -->
+        </div>
+    </div>
+
+    <!-- ID Card Wrapper Back Side -->
+    <div class="id-card-wrapper-back">
+        <!-- Card Background Back Side -->
+        <div class="id-card-background">
+            <div class="id-dark-pink-bottom"></div>
+            <div class="id-pink-bottom"></div>
+            <!-- <div class="id-light-pink-bottom"></div> -->
+        </div>
+
+        <!-- Content of ID Card Back Side -->
+        <div class="id-card-content">
+
+            <div class="qr-code">
+                <img src="img/qr.jpg" alt="Photo">
+            </div>
+
+            <!-- Emergency Contact Information -->
+            <div class="id-emergency-contact">
+                <!-- <strong>Emergency Contact:</strong> Abc<br> -->
+                <!-- <strong>Phone:</strong> --> +000 98765 4321 
+            </div>
+
+            <!-- Blood Group -->
```

Comment:
id-blood-group class is missing if you don't have content no issue but do not remove the class